src/user/ci-skeleton.rst

```diff
@@ -55,7 +55,7 @@ not running ``myproj`` CI as a regular feedstock. The ``.gitignore`` is
 modified to not accidentally add unwanted conda-smithy temporary files
 to your repository.
 
-Also the steps that the ``ci-skeleton`` spits out are very important
```

Comment:
I think, 'spits out' is correct. 
'The steps that ci-skeleton spits out..' means the steps this command displays.